src/test/infra/lttng/test_latency_definitions.py

```diff
@@ -1256,6 +1298,55 @@ def test_inter_proc(
 
         assert df.equals(df_expect)
 
+    def test_intra_proc_empty_data(
+        self,
+        mocker,
+        create_lttng,
+        create_publisher_lttng,
+        setup_bridge_get_publisher,
+        create_publisher_struct,
+        bridge_setup_get_callback,
+        create_subscription_lttng,
+        create_subscription_struct,
+        create_comm_struct
+    ):
+        pub_handle = 7
+        callback_obj = 12
+        callback_obj_intra = 5
+        subscription_handle = 8
+
+        data = Ros2DataModel()
+        data.finalize()
+
+        publisher = create_publisher_struct()
+        subscription = create_subscription_struct()
+        callback = subscription.callback
+        communication = create_comm_struct(publisher, subscription)
+        publisher_lttng = create_publisher_lttng(pub_handle)
+        sub_cb_lttng = create_subscription_lttng(callback_obj, subscription_handle, callback_obj_intra)
```

Comment:
This line exceeds 100 characters.

Reply:
addressed [65719fb](https://github.com/tier4/CARET_analyze/pull/210/commits/65719fb1c28ad9334f00127e17d4fa77468c4341)